src/app/loans/loans-view/loan-account-actions/add-collateral/add-collateral.component.ts

```diff
@@ -0,0 +1,75 @@
+/** Angular Imports. */
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormBuilder, Validators } from '@angular/forms';
+import { ActivatedRoute, Router } from '@angular/router';
+
+/** Custom Services. */
+import { LoansService } from 'app/loans/loans.service';
+
+/**
+ * Add Collateral component.
+ */
+@Component({
+  selector: 'mifosx-add-collateral',
+  templateUrl: './add-collateral.component.html',
+  styleUrls: ['./add-collateral.component.scss']
+})
+export class AddCollateralComponent implements OnInit {
+
```

Comment:
Implement `@Input()` to receive the data in the component.

Reply:
Done